README.md

```diff
@@ -13,7 +13,24 @@
 
 This is a simple experimental binary artifacts manager.
 
-TBD...
+## How does it work
+
+Artipie uses external server implementation to start itself,
```

Comment:
@g4s8 maybe we should add another puzzle for starting Artipie server?

Reply:
@iakunin we didn't decide how exactly the server will be started, I think it will be don in scope of another ticket